defaults/main.yml

```diff
@@ -30,7 +30,7 @@ clickhouse_service_state: started
 clickhouse_log_output: "{% if clickhouse_log_file is defined %} file:{{ clickhouse_log_file }} {% else %} journal {% endif %}"
 
 clickhouse_private_tmp: false
-clickhouse_limit_NO_FILE: 500000
+clickhouse_limit_no_file: 500000
```

Comment:
Personally I would rather keep the old naming to be consistent with Ansible naming conventions

Reply:
wow, i misread, marking as resolved :)